src/data/registration/registration_rollback.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/sh
+
+# This script rolls back the registration status on the server
```

Comment:
please document that script have also one parameter

Reply:
Added.